docs/server-side-rendering.md

```diff
@@ -3,7 +3,7 @@ Server-side Rendering
 
 Server-side rendering is great for SEO, as well as progressive enhancement. When rendering on the server, we have a special set of constraints that we need to follow when building components and libraries.
 
-tl;dr: Don't depend on the DOM/BOM; make sure your initial render is synchronous; don't mutate class variables; add the `/** @ssr-ready **/` pragma.
+tl;dr: Don't depend on the DOM/BOM; make sure your initial render is synchronous; don't mutate class variables; add a test to `renderToString` your server-side rendered page.
```

Comment:
We don't, unfortunately. This might be a bit hard to implement since right now the only way to reliably tell something is SSR'ed is to look at the route definitions module (and check for the presence of a `router` arg and a `makeLayout` middleware), and then to check which components are being rendered for it...